### PR TITLE
462 reference modal

### PR DESCRIPTION
**Pull request by ilya-zhuravel**

### CODE REVIEW

File: src/app/assets/drag-indicator-grey.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (NuclearRedeye) [inline src/app/assets/drag-indicator-grey.svg:1]: I assume that Nick D supplies you with these? If so, just need to make sure that he has either created them or sourced them from somewhere that allows us to use/distribute them.

File: src/app/containers/reference-form-dialog/index.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    if (isNewReference) {
+      dispatch(manuscriptActions.addReferenceAction(userReference));
+    } else {
+      dispatch(manuscriptActions.updateReferenceAction(userReference));
+    }
+    closeDialog();
+  }, [userReference, closeDialog, dispatch, isNewReference]);
+
+  const handleReject = useCallback(() => {
+    setConfirmSnow(false);
```
Comment (NuclearRedeye) [inline src/app/containers/reference-form-dialog/index.tsx:61]: Typo? I guess it's no big deal but would be nice if it was corrected to be I assume 'setConfirmShow'

File: src/app/containers/reference-form-dialog/reference-authors-list.tsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+      value={refAuthor['lastName']}
+      onChange={handleFormChange}
+    />
+    <TextField
+      fullWidth
+      name="firstName"
+      label="Given name"
+      InputLabelProps={labelProps}
+      variant="outlined"
+      value={refAuthor['firstName']}
```
Comment (NuclearRedeye) [inline src/app/containers/reference-form-dialog/reference-authors-list.tsx:57]: This always is the first authors initial, but this doesn't look incorrect hence I guess the error is where you're extracting the info from the XML into the store when loading.

File: src/app/assets/drag-indicator-grey.svg (hunk 1 above)
Comment (ilya-zhuravel) [reply src/app/assets/drag-indicator-grey.svg:1]: It's the same icon as the drag handle on the sort authors list but in different colour

**Pull request merged**